target/classes/META-INF/resources/home_page.html

```diff
@@ -40,6 +40,9 @@
                         <li class="nav-item dropdown">
                             <a class="nav-link" href="\item_pages\general.html">General Items</a>
                         </li>
+                        <li class="nav-item-dropdown">
+                            <a class="nav-link" href="\item_pages\donation-form.html">Donation Form</a>
```

Comment:
To me it seems like pages under "item_pages" should be pages that show the items being donated, so the donation form page wouldn't be there. What do you think?

Reply:
I changed it so it's now under donation_page. I feel like this also makes it more organized to have all of the donation page files under their own folder.